src/utils/tokens.test.ts

```diff
@@ -1,50 +1,12 @@
 import * as common from '@protocolink/common';
 import { expect } from 'chai';
-import { get1InchTokens, getDefaultTokenListUrls, getMetisTokens, getTokenList } from './tokens';
+import { getUnifiedTokens } from './tokens';
 
 describe('Test tokens', function () {
   common.networks.forEach(({ chainId }) => {
     it(`network: ${common.toNetworkId(chainId)}`, async function () {
-      const tokens = chainId === common.ChainId.metis ? await getMetisTokens() : await get1InchTokens(chainId);
+      const tokens = await getUnifiedTokens(chainId);
       expect(tokens).to.have.lengthOf.above(0);
     });
   });
 });
-
-const chainsToTest = [
-  common.ChainId.mainnet,
-  common.ChainId.optimism,
-  common.ChainId.polygon,
-  common.ChainId.base,
-  common.ChainId.arbitrum,
-  common.ChainId.avalanche,
-];
-
-describe('Test getDefaultTokenListUrls', function () {
```

Comment:
Can we move the test to `protocolink-logics/src/logics/paraswap-v5/logic.swap-token.test.ts` rather than deleting?